### DISCUSSION TITLE
An unhandlable issue was found during Windows compilation

### DISCUSSION THREAD

**Discussion by which-biscuits** [Unipisa/Simu5G#216] [Q&A]
Hello, when I added simu5G to the Windows version of Omnetpp, the compilation process kept reporting errors. 
The error message is:
Llvm ar.exe: error: unable to load '\/ Out /lang release /src /libsimu5g. dll. a ': No such file or directory.
I would like to ask if you have ever encountered a similar problem.

**Comment by giovanninardini** [comment]
Hello, 

how did you "added Simu5G to the Windows version of Omnetpp"?
Did you that from the OMNeT++ IDE (help -> install simulation models)? Did you install INET 4.5 first?

**Comment by which-biscuits** [comment]
hello,
Yes, I tried both "INET4.4+Simu5G1.2.1" and "INET4.5+Simu5G1.2.2" on Windows, and tried importing from omnetpp and downloading source code directly. But they all have this kind of error.
Thank you for your reply!
